src/RA_httpthread.cpp

```diff
@@ -252,31 +247,74 @@ void RAWeb::SetUserAgentString()
 #ifndef NTSTATUS
     #define NTSTATUS long
 #endif
-    using fnRtlGetVersion = NTSTATUS(NTAPI*)(PRTL_OSVERSIONINFOEXW lpVersionInformation);
-    auto RtlGetVersion = (fnRtlGetVersion)GetProcAddress(GetModuleHandleW(L"ntdll.dll"), "RtlGetVersion");
-    if (RtlGetVersion)
     {
-        RTL_OSVERSIONINFOEXW osVersion = {};
+        // needs to be XP or higher no point in continuing
```

Comment:
We still need to construct a User Agent regardless of the Windows version. 

What was wrong with the code before? On XP (or before XP), the GetModuleHandle or the GetProcAddress would fail and we'd simply skip over the NT version section of the User Agent.

Reply:
Nothing was wrong but if we don't care about the windows version here it can go back